instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go

```diff
@@ -85,7 +85,7 @@ func (h *serverHandler) TagRPC(ctx context.Context, info *stats.RPCTagInfo) cont
 
 // HandleRPC processes the RPC stats.
 func (h *serverHandler) HandleRPC(ctx context.Context, rs stats.RPCStats) {
-	h.handleRPC(ctx, rs)
+	h.handleRPC(ctx, rs, true)
```

Comment:
I do not find it readable.

I propose the following

```suggestion
	isServer := true
	h.handleRPC(ctx, rs, isServer)
```